### PR TITLE
Fixed exc_bad_access

### PR DESCRIPTION
**Pull request by itruf**
When delegate or self already invalidated it caused crashes

### CODE REVIEW

File: AutoLayoutTextViews/ALAutoResizingTextView.m [hunk 1]
```diff
@@ -153,20 +153,23 @@ - (void)setNewHeightWithoutAnimation
 
 - (void (^)())animationBlock
 {
+  __weak ALAutoResizingTextView *weakSelf = self;
   return  ^{
-    if ([self.delegate respondsToSelector:@selector(textView:willChangeFromHeight:toHeight:)]) {
-      [self.delegate textView:self willChangeFromHeight:self.oldHeight toHeight:self.newHeight];
+    ALAutoResizingTextView *innerSelf = weakSelf;
```
Comment (JRG-Developer) [inline AutoLayoutTextViews/ALAutoResizingTextView.m]: Change    
    `ALAutoResizingTextView *innerSelf = weakSelf;`

to
    `__strong ALAutoResizingTextView *strongSelf = weakSelf;`

per common conventions, and likewise rename other `innerSelf` calls to `strongSelf`.

File: AutoLayoutTextViews/ALAutoResizingTextView.m [hunk 2]
```diff
@@ -153,20 +153,23 @@ - (void)setNewHeightWithoutAnimation
...
 }
 
 - (void(^)(BOOL))completionBlock
 {
+  __weak ALAutoResizingTextView *weakSelf = self;
   return ^(BOOL finished) {
-    
-    if ([self.delegate respondsToSelector:@selector(textView:didChangeFromHeight:toHeight:)]) {
-      [self.delegate textView:self didChangeFromHeight:self.oldHeight toHeight:self.newHeight];
+    ALAutoResizingTextView *innerSelf = weakSelf;
```
Comment (JRG-Developer) [inline AutoLayoutTextViews/ALAutoResizingTextView.m]: Likewise here.

**Comment by JRG-Developer** [comment]
Thanks for the PR!

Please make the minor changes requested, and I'll be happy to accept it. 👍

**Comment by JRG-Developer** [comment]
Looks good! Merged.

Thanks. 😄

**Comment by itruf** [comment]
Fine, thx for fast acceptance :)

**Comment by JRG-Developer** [comment]
I've also pushed this to CocoaPods as version `1.2.6` (in case you use this).

👍

**Comment by itruf** [comment]
Yep. I’ve already uploaded build with my fork to app store, but I will change it.

**Pull request merged**